interface-definitions/interfaces-pppoe.xml.in

```diff
@@ -50,6 +50,19 @@
               <constraintErrorMessage>Host-uniq must be specified as hex-adecimal byte-string (even number of HEX characters)</constraintErrorMessage>
             </properties>
           </leafNode>
+          <leafNode name="holdoff">
+            <properties>
+              <help>Delay before re-dial to the access concentrator when PPP session terminated by peer (in seconds)</help>
+              <valueHelp>
+                <format>u32:1-86400</format>
+                <description>Holdoff time in seconds, default 30</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-86400"/>
+              </constraint>
+              <constraintErrorMessage>Holdoff must be in range 1 to 86400</constraintErrorMessage>
+            </properties>
+          </leafNode>
```

Comment:
If there is a default, please use <defaultValue> XML node

Reply:
The default value is defined by pppd itself.

Reply:
Yeah, but it would not harm to transfer this to the CLI sonthe user will see the auto generated defaultValue from XML.